src/Algebra/Linear/Bundles.agda

```diff
@@ -0,0 +1,95 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some bundles of linear algebraic structures.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Linear.Bundles where
+
+import Algebra.Module.Morphism.Structures as MorphismStructures
+
+open import Algebra        using (CommutativeRing)
+open import Algebra.Module using (Module)
+open import Algebra.Module.Construct.TensorUnit using (⟨module⟩)
+open import Algebra.Module.Morphism.Bundles
+open import Data.List      using (List; foldr)
+open import Data.Product
+open import Function
+open import Level          using (Level; _⊔_; suc)
+open import Relation.Binary
+open import Relation.Binary.ExtensionalEquivalence
+open import Relation.Binary.Reasoning.MultiSetoid
+
+private
+  variable
+    a b c : Level
+    A : Set a
+    B : Set b
+    C : Set c
+
+-- Abstract vector spaces.
+--
+-- A "vector space" is a Module with a commutative, homomorphic inner
+-- product and a complete set of building blocks for mapping the space.
+record VectorSpace
```

Comment:
Can we follow the standard pattern in `Algebra` and split this out into a `IsVectorSpace` structure in `Algebra.Linear.Structure` and `VectorSpace` bundle in `Algebra.Linear.Bundle`?

Reply:
Yep. Arriving w/ next push.